README.md

```diff
@@ -1,7 +1,8 @@
 [![][ButlerImage]][website] 
 
+
 # About
-In a nutshell, Jenkins CI is the leading open-source continuous integration server. Built with Java, it provides over 1000 plugins to support building and testing virtually any project.
+In a nutshell, Jenkins CI is the leading open-source continuous integration server. What this means is Jenkins is like the butler you never had-spill something or get a bug in your code-well Jenkins will help you clean it up. Now you'll know were a bug was most likely introduced so you can find it, fix it and get back to your project.Built with Java, it provides over 1000 plugins to support building and testing virtually any project.
```

Comment:
Typo: were a bug -> where a bug.

Not convinced anyway :-).

And if there's something to fix, IMO that would be more the "continuous integration server" part that needs addressing (Jenkins is not only a CI server, as we're trying to advertise...)